docs/recipes/plugin/readme.md

```diff
@@ -0,0 +1,18 @@
+# Lighthouse plugin recipe
+
+## Contents
+- `lighthouse-plugin-preload-as` - the plugin module
```

Comment:
Another thing I'm noticing... the Plugin is really about describing the category, not one of the two audits within it.  

This category is titled 'My Plugin Category'... but let's pretend it was called 'Example Plugin'.   I would expect the folder it's in to match the name of the category.. a la `lighthouse-plugin-example-plugin`

Reply:
renamed `lighthouse-plugin-example`